app/services/notices/setup/send-notice.service.js

```diff
@@ -59,6 +63,19 @@ async function _recordResult(sendResult) {
   await NotificationModel.query().patch({ pdf, plaintext, notifyError, notifyId, notifyStatus, status }).findById(id)
 }
 
+async function _sendAlternateNotice(notice) {
+  const { id: noticeId, subtype } = notice
+
+  if (subtype === 'returnInvitation') {
+    const { notice: alternateNotice, notifications } = await CreateAlternateNoticeService.go(notice)
+
+    if (notifications.length > 0) {
+      await _sendNotifications(notifications, alternateNotice.referenceCode)
+      await _updateFailedEmailInvitations(noticeId)
+    }
+  }
```

Comment:
Our normal pattern is to use a guard clause and bail early.


```suggestion
  if (subtype !== 'returnInvitation') {
    return
  }

  const { notice: alternateNotice, notifications } = await CreateAlternateNoticeService.go(notice)

  if (notifications.length > 0) {
    await _sendNotifications(notifications, alternateNotice.referenceCode)
    await _updateFailedEmailInvitations(noticeId)
  }
```